### ISSUE TITLE
SearchView button duplicate when back button press

### ISSUE THREAD

**Issue by KushiRangika** [Blasanka/StudyMate#1]
In Task fragment, completed task, after add a task SearchView button duplicate when back button press

**Comment by Blasanka** [comment]
Yeah, I checked that out, and currently Im busy with Quiz, I will get into it soon, thanks Kushi
